assets/src/console/pages/products/ListPage.tsx

```diff
@@ -2,19 +2,20 @@ import * as React from "react";
 import { useRouteMatch } from "react-router-dom";
 import { Helmet } from "react-helmet";
 import {
-  TODO,
   PageTitle,
   Loader,
   ErrorAlert,
   Link,
   Warning,
-  P
+  P,
+  styled
 } from "@pay/web";
 
 import {
   GatewayAccountFragment,
   useGetProductsQuery
 } from "../../__generated__/graphql";
+import { SidebarLayout } from "../../../checkout/components/Split";
```

Comment:
`console` app shouldn't be importing things from `checkout` app

Reply:
I'll move this stuff to shared components in future PR. I put it in the wrong spot to begin with; don't want to mix concerns in PR